app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,73 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      @user = current_user
+      @tweets = Tweet.all
+      erb :"views/tweets/tweets"
+     else
+       redirect to "/login"
+     end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      @user = current_user
+      erb :'views/tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/new/tweet' do
```

Comment:
Typically, tweet creation would be posted to `/tweets` instead of `/tweets/new`.